src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -511,8 +511,19 @@ private void LogLocals(IEnumerable<ILocalSymbol> locals, string header = "Locals
 
         private void LogLoopStatementHeader(ILoopOperation operation)
         {
-            var kindStr = $"{nameof(LoopKind)}.{operation.LoopKind}";
-            LogString($" ({kindStr})");
+            var propertyStringBuilder = new StringBuilder();
+            propertyStringBuilder.Append(" (");
+            propertyStringBuilder.Append($"{nameof(LoopKind)}.{operation.LoopKind}");
+            if (operation.ContinueLabel != null)
+            {
+                propertyStringBuilder.Append(", ContinueLabel");
```

Comment:
>propertyStringBuilder.Append(", ContinueLabel"); [](start = 16, length = 48)

I think we should at least include label's name as in VisitBranch(IBranchOperation operation).

Reply:
Also, given that the names of the labels often not unique, I think we should assign some kind of unique id to each label and include it along with the name or instead of the name. This applies to all nodes that expose label symbols. Probably can be done separately to avoid massive baseline changes for tests unrelated to the scenarios primarily targeted by this PR. 

---
In reply to: [169866369](https://github.com/dotnet/roslyn/pull/24979#discussion_r169866369) [](ancestors = 169866369)

Reply:
I switched to generating ids, as C#'s generated label names change every run.

---
In reply to: [169867028](https://github.com/dotnet/roslyn/pull/24979#discussion_r169867028) [](ancestors = 169867028,169866369)